### DISCUSSION TITLE
[Curreny] Option to hide the currency

### DISCUSSION THREAD

**Discussion by darthf1** [formkit/formkit#1409] [Ideas]
I know, it doesn't make sense to remove the currency from the currency input. But I really like the number formatting part of this component. In my project, I currently have two currency components:
- One displaying a money value with the currency, using formkits currency component.
- One displaying a money value without the currency, using `vue-currency-input`.

It would be great if I can use formkits currency component for both cases.

If I omit the `currency` prop (or set it to undefined), it defaults to `usd`. Maybe a `false` value can be allowed, which hides the currency from the output?

**Comment by andrew-boyd** [comment]
This is a good idea! Should be doable if we expose a prop to allow users to modify the values being passed to the INTL formatter.

**Reply by andrew-boyd** [reply]
I've created an issue from this discussion. #1410
